spec/features/data_access/data_access_workflow_spec.rb

```diff
@@ -39,10 +40,13 @@
     end
 
     it "forces the user to login before showing data access page", intermittent: true do
-      screenshot_path = "./tmp/screenshots/debug-#{Time.now.to_i}.png"
-      expect(page).to have_content('EOSDIS Earthdata Login'), lambda {
-        "Expect to see 'EOSDIS Earthdata Login' on the page. #{page.save_screenshot(screenshot_path)}"
-        Rails.logger.info Base64.encode64(File.open(screenshot_path, "rb").read)}
+      unless page.has_content?('EOSDIS Earthdata Login')
```

Comment:
To make sure that the user was logged in ('Manage user account' link will be there in this case) if the redirection finishes before CI looks at the page.

The removed part that base64 encode the screenshot indicates that the user has been redirected back to Earthdata Search already when the test expects the user being on Earthdata Login page.